src/matrix.js

```diff
@@ -136,6 +154,11 @@ module.exports.wrapRequest = function(wrapper) {
  * {@link module:scheduler~MatrixScheduler}.
  * @param {requestFunction} opts.request If not set, defaults to the function
  * supplied to {@link request} which defaults to the request module from NPM.
+ *
+ * @param {module:crypto.store.base~CryptoStore=} opts.cryptoStore
+ *    crypto store implementation. Calls the factory supplied to
```

Comment:
Should specify what the default is if the caller never explicitly calls `setCryptoStoreFactory`. Does it error out? In-memory?